CI-CD/GitHub/Mend CLI/Github_Advanced_Linux/mend-scan.yaml

```diff
@@ -0,0 +1,37 @@
+# ******** Mend Unified CLI Template for Github Actions ********
+#
+# You may wish to alter this file to override the build tool and Mend scanning technologies.
+#
+# For more configuration options, please check the technical documentation portal:
+# 📚 https://docs.mend.io/bundle/integrations/page/scan_with_the_mend_cli.html
+#
+# ******** Description ********
+# mend dep will automatically use package managers and file system scanning to detect open source components.
+# mend code will automatically detect languages and frameworks used in your projects to scan for code weaknesses.
+
+# If you are NOT using a service user, and have multiple organizations, don't forget to call the scope -s parameter to set the organizatio
+
+name: Mend
+
+on:
+  push:
+    branches: [ main ]
+
+
+jobs:
+
+  call-template:
+    uses: mend-examples/mend-toolkit/.github/workflows/mend-scan-template.yaml@scan-templates
```

Comment:
Add a comment here to explain this is a public template which can be used in testing, but for production should be copied to your own repo.

Reply:
Thanks @lukebrogan-mend , added